dsgrid/registry/registry_manager_base.py

```diff
@@ -526,9 +526,6 @@ def remove(self, config_id):
         # TODO: Do we want to handle specific versions? This removes all configs.
 
     def _remove(self, config_id):
-        if not self.offline_mode:
-            # TODO: DSGRID-145
-            raise Exception("sync-push of config removal is currently not supported")
```

Comment:
This was a bug. If you hit an error while registering something in online mode, you would get this. Your local registry would be left in an odd state. In theory, it would get cleaned up in the next sync. However, I think this is the best thing to do.
- `remove` is not available in the CLI
- We don't perform any syncing on error.